index.html

```diff
@@ -183,7 +184,45 @@ <h4>Skills</h4>
                 </ul>
             </div>
         </div> 
+    </div>
     </section>
-</section>
+    <footer class="form">
+        <h3>
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h3>
+        <div class="form-section">
+            <form action="https://formspree.io/f/myyvjjng" method="post" id="form">
+                <div class="form-input">
+                    <input type="text"  id="name" name="user_name" placeholder="Full name" maxlength="30" required>
+                    <input type="email"  id="email" name="user_email"  placeholder="Email address" required>
+                    <textarea type="text"  id="message" name="user_message"  maxlength="500" required></textarea>
+                </div>
+                <div class="back-img4"></div>
+                <div class="get-in-touch-btn">
+                    <button class="form-btn" type="submit">Get in touch </button>
+                </div>
+            
+            </form>
+        </div>
+            <div class="contact-social">
+                <ul class="socialt">
+                    <li> <a href="#"><img src="image/twitter.svg" /></a> &nbsp; &nbsp; </li>
+                    <li> <a href="#"><img src="image/vector.png" /></a> </li>
+                    <li> <a href="#"><img src="image/Vector (1).png" /></a> &nbsp; &nbsp; </li>
+                    <li> <a href="#"><img src="image/medium.svg" /></a> &nbsp; &nbsp; </li>
+                    <li> <a href="#"><img src="image/github.svg" /></a> &nbsp; &nbsp; </li>
+                    <li> <a href="#"><img src="image/angellist.svg" /></a> &nbsp; </li>
+                 </ul> 
+                 <div class="footer-email">
+                    <a href="mailto:hiwina@gmail.com" id="">hiwina@gmail.com</a>                
+               </div>
+            </div>
+            
+            <div>
+                <img src="image/Light.png" alt="footer image" id="footer-image">
+
+            </div>
```

Comment:
- According to the comment above. Please wrap this information inside a footer tag. :+1: :smiley: :smiley: 

- Kindly make sure that all your social media icons are rendering correctly. There is an issue with the LinkedIn icon please fix it. :eyes: :eyes: 

![image](https://user-images.githubusercontent.com/98363075/194059022-76a16b81-744b-4e42-88ee-32d406fd5e7b.png)